### PR TITLE
Add tls ssl key endpoints to traffic_ops_golang.

### PR DESCRIPTION
**Pull request by jrushford**
This PR adds 3 API endpoints to the traffic_ops_golang, from CDN-750.  The endpoints add certificates to RIAK and  fetch certificates from RIAK by hostname and xmlid.  The add function verifies and orders the certificate chain before storing to RIAK.

Talked to @dewrich and since these endpoints do not yet implement tenancy checks they are marked as "-wip" in the routes.go.  Will implement tenancy at a later date.  Want to get this in to avoid merge conflicts going forward.

### CODE REVIEW

**Comment by asfgit** [comment]
SUCCESS 
 No test results found.
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/664/
Test PASSed.

**Comment by asfgit** [comment]
SUCCESS 
 No test results found.
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/666/
Test PASSed.

File: traffic_ops/traffic_ops_golang/routes.go [hunk 1]
```diff
@@ -77,6 +77,10 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		{1.2, http.MethodPost, `servers/{id}/deliveryservices$`, assignDeliveryServicesToServerHandler(d.DB), auth.PrivLevelOperations, Authenticated, nil},
 		{1.2, http.MethodGet, `servers/{host_name}/update_status$`, getServerUpdateStatusHandler(d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
 
+		//SSLKeys deliveryservice endpoints here that are marked  marked as '-wip' need to have tenancy checks added
```
Comment (mitchell852) [inline traffic_ops/traffic_ops_golang/routes.go]: i think if these endpoints are simply replacing existing perl endpoints they should be 1.2 not 1.3

**Comment by asfgit** [comment]
SUCCESS 
 No test results found.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/623/

**Comment by asfgit** [comment]
SUCCESS 
 No test results found.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/625/

**Comment by asfgit** [comment]
SUCCESS 
 No test results found.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/635/

**Comment by asfgit** [comment]
FAILURE
 No test results found.
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/676/
Test FAILed.

File: lib/go-tc/deliveryservice_ssl_keys.go [hunk 2]
```diff
@@ -15,28 +15,72 @@ package tc
...
-	Hostname        string                            `json:"hostname"`
-	Country         string                            `json:"country"`
-	State           string                            `json:"state"`
-	Version         string                            `json:"version"`
-	Certificate     DeliveryServiceSSLKeysCertificate `json:"certificate"`
-}
-
 // DeliveryServiceSSLKeysCertificate ...
 type DeliveryServiceSSLKeysCertificate struct {
 	Crt string `json:"crt"`
```
Comment (dewrich) [inline lib/go-tc/deliveryservice_ssl_keys.go]: Just a nit but should `Crt` be `CRT`?

**Pull request merged**